image_rotate/src/image_rotate_node.cpp

```diff
@@ -128,13 +106,31 @@ ImageRotateNode::ImageRotateNode(const rclcpp::NodeOptions & options)
       source_vector_.vector.x = config_.source_x;
       source_vector_.vector.y = config_.source_y;
       source_vector_.vector.z = config_.source_z;
-      if (subscriber_count_) {  // @todo: Could do this without an interruption at some point.
-        unsubscribe();
-        subscribe();
-      }
```

Comment:
this is unneeded - none of the parameters that are able to be updated would impact how the subscribers would actually be setup